habito/habito.py

```diff
@@ -140,6 +140,41 @@ def add(name, quantum, units):
                       created_date=datetime.now(), quantum=quantum,
                       units=units, magica="")
 
+@cli.command()
+@click.argument("id", type=click.INT)
+@click.option('--name', '-n', help="The new name (leave empty to leave unchanged)")
+@click.option('--quantum', '-q', help="The new quantum (leave empty to leave unchanged)", type=click.FLOAT)
+def edit(id, name, quantum):
```

Comment:
I think, now that I think more about it, that there are a number of scenarios:

1. a typo; the quantum was set incorrectly;
2. overestimating one's ability and setting a more realistic quantum;
3. growth. After having achieved a habit at a certain quantum, it may be desirable to 'up the quantum', e.g. after having written 200 words each day for 5 days, you might want to write 300 words each day.

Scenario 1 and 2 are not problematic, I think. Scenario 3 is particularly interesting, because it's often an inherent part of building habits: you want to start out small, increment the quantum in small steps to finally get to the ultimate goal. It might be useful to build this into habito: after an x days streak, propose to up the quantum by 50% (or by whatever value provided by the user). Another option is to set this up straight away; e.g. when adding a habit, set up the goal for this day, the ultimate goal and the nr of days you want to give yourself. E.g. write 100 words today, ultimate goal is 1000 words; give yourself 100 days -> increment by 10 words per day. There are many options here; my ideas are just to get the discussion going.

There of course is always the option of misuse, but as I mentioned in #7, I'm not sure whether it's up to us to prevent this. We can always add a warning (are you sure you want to do this?), but I'm not sure whether it's possible to prevent this without limiting the user's ability to edit for genuine (scenario 1 and 2) reasons.